CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [0.4.0] - 2024-07-26
```

Comment:
I think that if we don't move `OxigraphStore` the changes might be a 0.3.8 release

Reply:
Agree, let's keep it as it was and stick to version 0.3.x

Reply:
set version to 0.3.8